NOTICE.txt

```diff
@@ -85,6 +80,12 @@ This product contains a modified part of Netty, distributed by Netty.io:
   * License: licenses/LICENSE.netty.al20.txt (Apache License v2.0)
   * Homepage: http://netty.io/
 
+This product contains a modified part of Rolling-Metrics, distributed by Vladimir
+Bukhtoyarov:
+
+  * License: licenses/LICENSE.rolling-metrics.al20.txt (Apache License v2.0)
+  * Homepage: https://github.com/vladimir-bukhtoyarov/rolling-metrics
```

Comment:
Wished to reuse this as-is, but the implementation had to stay independent from both metric libraries.